From abcbd17f43e945c5b2813f2ebeb801b7233d0ae2 Mon Sep 17 00:00:00 2001
From: Nick Xie <nick@khadas.com>
Date: Wed, 27 May 2020 13:57:05 +0800
Subject: [PATCH] 1111

Signed-off-by: Nick Xie <nick@khadas.com>
---
 scripts/package/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 2aaf7beb04ce..f97369626461 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -85,7 +85,7 @@ create_package() {
 
 	# Create postinstall script for headers
 	if [ "$3" = "headers" ]; then
-		echo "cd /usr/src/linux-headers-$version; echo \"Compiling headers - please wait ...\"; find -type f -exec touch {} +;make -s scripts >/dev/null; make -s M=scripts/mod/ >/dev/null" >> $pdir/DEBIAN/postinst
+		echo "cd /usr/src/linux-headers-$version; echo \"Compiling headers - please wait ...\"; find -type f -exec touch {} +;make -j4 -s scripts >/dev/null; make -j4 -s M=scripts/mod/ >/dev/null" >> $pdir/DEBIAN/postinst
 		echo "exit 0" >> $pdir/DEBIAN/postinst
 		chmod 775 $pdir/DEBIAN/postinst
 	fi
-- 
2.17.1

