From c719420545b826eed78acb016205bbfa59ba5a1f Mon Sep 17 00:00:00 2001
From: Nick Xie <nick@khadas.com>
Date: Fri, 19 Feb 2021 18:54:09 +0800
Subject: [PATCH] fix debian build

Signed-off-by: Nick Xie <nick@khadas.com>
---
 scripts/package/builddeb | 2 --
 1 file changed, 2 deletions(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index bb43f358ff26..08a4a9c6b343 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -342,8 +342,6 @@ check_and_unmount (){
 		fi
 	done
 
-	deploy_libc_headers debian/linux-libc-dev
-	create_package linux-libc-dev debian/linux-libc-dev
 	bootfstype=\$(blkid -s TYPE -o value \$boot_partition)
 	if [ "\$bootfstype" = "vfat" ]; then
 		# we have to keep it mounted! umount /boot
-- 
2.25.1

