From 646417081d3e35af6f8141b3372bb8bf6297eb35 Mon Sep 17 00:00:00 2001
From: Nick Xie <nick@khadas.com>
Date: Thu, 26 Dec 2019 00:15:18 +0800
Subject: [PATCH] builddeb: update dtb file for SD images

Signed-off-by: Nick Xie <nick@khadas.com>
---
 scripts/package/builddeb | 30 +++++++++++++++++++++++++++++-
 1 file changed, 29 insertions(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 7bdac12..47d742e 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -49,9 +49,37 @@ create_package() {
 	    echo "exit 0" >> $pdir/DEBIAN/preinst
 	    chmod 775 $pdir/DEBIAN/preinst
 
+		echo "#!/bin/bash" >> $pdir/DEBIAN/postinst
+		echo "source /etc/fenix-release" >> $pdir/DEBIAN/postinst
 	    echo "if [ -d /boot/dtb-$version.old ]; then rm -rf /boot/dtb-$version.old; fi" >> $pdir/DEBIAN/postinst
 	    echo "ln -sf dtb-$version /boot/dtb > /dev/null 2>&1 || mv /boot/dtb-$version /boot/dtb" >> $pdir/DEBIAN/postinst
-	    echo "exit 0" >> $pdir/DEBIAN/postinst
+		cat <<-EOF >> $pdir/DEBIAN/postinst
+			case \$BOARD in
+				VIM1)
+					DTB=meson-gxl-s905x-khadas-vim.dtb
+					;;
+				VIM2)
+					DTB=meson-gxm-khadas-vim2.dtb
+					;;
+				VIM3)
+					DTB=meson-g12b-a311d-khadas-vim3.dtb
+					;;
+				VIM3L)
+					DTB=meson-sm1-khadas-vim3l.dtb
+					;;
+				Edge)
+					DTB=rk3399-khadas-edge.dtb
+					;;
+				Edge-V)
+					DTB=rk3399-khadas-edge-v.dtb
+					;;
+				Captain)
+					DTB=rk3399-khadas-edge-captain.dtb
+					;;
+			esac
+			cp /boot/dtb/\${VENDOR,,}/\$DTB /boot/dtb.img
+			exit 0
+		EOF
 	    chmod 775 $pdir/DEBIAN/postinst
 	fi
 
-- 
2.7.4

