From 663200efd8281d252176755903d0241190f70e6c Mon Sep 17 00:00:00 2001
From: Artem Lapkin <art@khadas.com>
Date: Tue, 20 Oct 2020 18:46:31 +0800
Subject: [PATCH 1/2] modify_gitignore_list

---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.gitignore b/.gitignore
index 6bec6b7c..d21ffdd3 100644
--- a/.gitignore
+++ b/.gitignore
@@ -5,6 +5,9 @@
 #
 # Normal rules (sorted alphabetically)
 #
+
+/bl31.c
+
 .*
 *.a
 *.asn1.[ch]
-- 
2.25.1

