From bac6431ae84095f4f132d17057904b9c305be93a Mon Sep 17 00:00:00 2001
From: Yan <yan-wyb@foxmail.com>
Date: Mon, 22 Nov 2021 10:49:26 +0800
Subject: [PATCH 65/70] add npu to staging

Signed-off-by: Yan <yan-wyb@foxmail.com>
---
 drivers/staging/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/staging/Makefile b/drivers/staging/Makefile
index 3ffb35ccfae2..a162102358eb 100644
--- a/drivers/staging/Makefile
+++ b/drivers/staging/Makefile
@@ -34,3 +34,4 @@ obj-$(CONFIG_XIL_AXIS_FIFO)	+= axis-fifo/
 obj-$(CONFIG_FIELDBUS_DEV)     += fieldbus/
 obj-$(CONFIG_QLGE)		+= qlge/
 obj-$(CONFIG_WFX)		+= wfx/
+obj-$(CONFIG_ARCH_MESON) += npu/
-- 
2.25.1

