From 3e368ea45e46df2c11466878a00a44d8d14ee139 Mon Sep 17 00:00:00 2001
From: Zhang Ning <1408979+zhangn1985@user.noreply.gitee.com>
Date: Sun, 1 May 2022 10:19:03 +0800
Subject: [PATCH 68/70] export symbol max_pcie_mrrs

---
 drivers/pci/pci.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/pci.c b/drivers/pci/pci.c
index 2b98be55f6ab..b2fbb944d07e 100644
--- a/drivers/pci/pci.c
+++ b/drivers/pci/pci.c
@@ -120,6 +120,7 @@ enum pcie_bus_config_types pcie_bus_config = PCIE_BUS_DEFAULT;
 
 /* PCIe Max Read Request Size, default 4096 no limit */
 u16 max_pcie_mrrs = 4096;
+EXPORT_SYMBOL(max_pcie_mrrs);
 
 /*
  * The default CLS is used if arch didn't set CLS explicitly and not
-- 
2.25.1

