From a8a5bc91ab4e2b3eace412f4bbc0f443c2f234ee Mon Sep 17 00:00:00 2001
From: yan <yan-wyb@foxmail.com>
Date: Fri, 26 Mar 2021 14:31:38 +0800
Subject: [PATCH] fixup package debs clean error

Signed-off-by: yan <yan-wyb@foxmail.com>
---
 scripts/package/builddeb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 8328f32f361a..68a304e873c7 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -377,7 +377,7 @@ mkdir -p "$destdir"
 ln -sf "/usr/src/linux-headers-$version" "$kernel_headers_dir/lib/modules/$version/build"
 rm -f "$objtree/debian/hdrsrcfiles" "$objtree/debian/hdrobjfiles"
 
-(cd $destdir; make M=scripts clean)
+#(cd $destdir; make M=scripts clean)
 
 if [ "$ARCH" != "um" ]; then
 	create_package "$kernel_headers_packagename" "$kernel_headers_dir" "headers"
-- 
2.17.1

