From 9418458f16e46bd19c3e0516f21aa293c6bae376 Mon Sep 17 00:00:00 2001
From: Zhang Ning <1408979+zhangn1985@user.noreply.gitee.com>
Date: Sun, 1 May 2022 10:19:03 +0800
Subject: [PATCH] export symbol max_pcie_mrrs

---
 drivers/pci/pci.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/pci.c b/drivers/pci/pci.c
index 823827805752..97b6e1dec726 100644
--- a/drivers/pci/pci.c
+++ b/drivers/pci/pci.c
@@ -120,6 +120,7 @@ enum pcie_bus_config_types pcie_bus_config = PCIE_BUS_DEFAULT;
 
 /* PCIe Max Read Request Size, default 4096 no limit */
 u16 max_pcie_mrrs = 4096;
+EXPORT_SYMBOL(max_pcie_mrrs);
 
 /*
  * The default CLS is used if arch didn't set CLS explicitly and not
-- 
2.35.1

