diff -Nuar rkbin-b789f81.orig/RKBOOT/RK3399MINIALL_SPINOR.ini rkbin-b789f81/RKBOOT/RK3399MINIALL_SPINOR.ini
--- rkbin-b789f81.orig/RKBOOT/RK3399MINIALL_SPINOR.ini	2018-08-16 15:58:22.394235253 +0800
+++ rkbin-b789f81/RKBOOT/RK3399MINIALL_SPINOR.ini	2018-08-16 15:59:47.269179699 +0800
@@ -5,7 +5,7 @@
 MINOR=14
 [CODE471_OPTION]
 NUM=1
-Path1=tools/rk_tools/bin/rk33/rk3399_ddr_800MHz_v1.12.bin
+Path1=tools/rk_tools/bin/rk33/rk3399_ddr_800MHz_v1.15.bin
 Sleep=1
 [CODE472_OPTION]
 NUM=1
@@ -14,7 +14,7 @@
 NUM=2
 LOADER1=FlashData
 LOADER2=FlashBoot
-FlashData=tools/rk_tools/bin/rk33/rk3399_ddr_800MHz_v1.12.bin
+FlashData=tools/rk_tools/bin/rk33/rk3399_ddr_800MHz_v1.15.bin
 FlashBoot=tools/rk_tools/bin/rk33/rk3399_miniloader_spinor_v1.14.bin
 [OUTPUT]
-PATH=rk3399_loader_spinor_v1.12.114.bin
+PATH=rk3399_loader_spinor_v1.15.114.bin
